main.go

```diff
@@ -109,6 +113,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	// convert -skip flags to -ignore equivalents
+	for _, s := range skipExtensionFlags {
+		ignorePatterns = append(ignorePatterns, fmt.Sprintf("**/*.%s", s))
+	}
+	// verify that all ignorePatterns are valid
+	for _, p := range ignorePatterns {
+		if !doublestar.ValidatePattern(p) {
+			log.Fatalf("ignore pattern %q is not valid", p)
```

Comment:
```suggestion go
log.Fatalf("ignore file pattern %q is not valid", p)
```

Reply:
would it be clearer if we said "-ignore pattern is not valid" (adding the leading hyphen)?  Then it's clear that we're talking about the pattern that was passed to the `-ignore` flag.

Reply:
Yeah. I think adding the leading hyphen can be better. Without it, it felt like `ignore pattern <something>.....` becomes clearer only after you get to the end of the sentence.